### DISCUSSION TITLE
Toggle button help

### DISCUSSION THREAD

**Discussion by rogerlauermann** [Power-Components/livewire-powergrid#411] [Q&A]
Hi,
How i do a slide toggle button? In my case is a checkbox, what could be wrong?

![Screenshot from 2022-04-27 15-05-49](https://user-images.githubusercontent.com/56614166/165597445-1ea6fa27-8a8e-4f52-8f7a-3a6fa4bfaba7.jpg)

I have seen many examples on web using a slide button, but here is a checkbox shows up.

**Comment by CrazyBoy49z** [comment]
Maybe a custom check box on the screen can be replaced here resources/views/vendor/livewire-powergrid/components/checkbox-row.blade.php
